blockchain/serviceMetadata.go

```diff
@@ -76,13 +155,33 @@ type ServiceMetadata struct {
 	freeCallsAllowed      int
 }
 
+type Subscription struct {
+	PeriodInDays    int    `json:"periodInDays"`
+	CreditsInAGI    int    `json:"creditsInAGI"`
+	PlanName        string `json:"planName"`
+	LicenseCost     int    `json:"LicenseCost"`
```

Comment:
Need to check LicenseCost here as well...

Reply:
Corrected this Sir